### ISSUE TITLE
how to configure mel filter bank design domain

### ISSUE THREAD

**Issue by shipley223** [librosa/librosa#1833]
I want to design mel filter in wrapped main instead of linear domain, can librosa support this ?
[FilterBankDesignDomain — Domain in which filter bank is designed](https://ww2.mathworks.cn/help/audio/ref/designauditoryfilterbank.html?#mw_457bc5f3-f50a-44ee-8fee-29114a7d3d82)

**Comment by bmcfee** [comment]
Sorry, but i didn't understand what you're asking. Can you rephrase or give an example of what you want to do?
